readme.md

```diff
@@ -268,6 +268,16 @@ Returns a set of supported file extensions.
 
 Returns a set of supported MIME types.
 
+### FileType.addDetection(detection)
+
+Register a detection function. This function will be called after `file-type` and other detections complete.
+
+The `detection(tokenizer, fileType)` method has two arguments:
+- [tokenizer](https://github.com/Borewit/strtok3#tokenizer)
+- `fileType`, which is `undefined` or the return value of a detected file type.
+
+If the detector returns `undefined`, the `tokenizer.position` should be 0. That allows other detectors to parse the file. 
```

Comment:
Why cannot `file-type` reset the position automatically?

Reply:
Because underlying source can be a stream. A stream can only be read once.

Reply:
Then it should be made clear that `the `tokenizer.position` should be 0` should not be done if it's a stream.